features/options/contract/version.feature

```diff
@@ -6,13 +6,6 @@ Feature: osrm-contract command line options: version
 
     Background:
         Given the profile "testbot"
-    
-    Scenario: osrm-contract - Version, short
```

Comment:
Because of the ambiguity between a `-v` flag to mean either `--version` or `--verify-weights` we're removing the short option entirely, and therefore don't need to test for it.